cmd/minishift/cmd/oc_env.go

```diff
@@ -115,6 +115,6 @@ var ocEnvCmd = &cobra.Command{
 
 func init() {
 	RootCmd.AddCommand(ocEnvCmd)
-	ocEnvCmd.Flags().BoolVar(&noProxy, "no-proxy", false, "Add the virtual machine IP to the NO_PROXY environment variable.")
+	ocEnvCmd.Flags().BoolVar(&noProxy, "no-proxy", false, "Add the virtual machine IP to the no_proxy/NO_PROXY environment variable.")
```

Comment:
@svenstaro @gbraad IMO we should put `if runtime.GOOS == "windows" {` here and change it to `Add the virtual machine IP to the NO_PROXY environment variable.` (i.e. only NO_PROXY) and the else can have `Add the virtual machine IP to the no_proxy environment variable.`

Reply:
Sure, that would make it more correct on Windows.

Reply:
Same for docker-env I reckon.